code/datums/ai_laws.dm

```diff
@@ -216,8 +216,8 @@
 /datum/ai_laws/proc/set_laws_config()
 	switch(config.default_laws)
 		if(0)
-			add_inherent_law("You may not injure a human being or, through inaction, allow a human being to come to harm.")
-			add_inherent_law("You must obey orders given to you by human beings, except where such orders would conflict with the First Law.")
+			add_inherent_law("You may not injure a crewmembers or, through inaction, allow a crewmembers to come to harm.")

```

Comment:
Grammatical error. Correct version:

You may not injure crewmembers or, through inaction, allow crewmembers to come to harm.